spec/controllers/idv/in_person/verify_info_controller_spec.rb

```diff
@@ -115,6 +115,34 @@
         )
       end
     end
+
+    context 'when pii_from_user' do
```

Comment:
Is this missing a words?

```suggestion
    context 'when pii_from_user is nil' do
```